Sourcery/Generating/Template/JavaScript/JavaScriptTemplate.swift

```diff
@@ -38,7 +38,13 @@ final class JavaScriptTemplate: Template {
 
         let include: @convention(block) (String) -> String? = { [unowned self] path in
             let path = self.sourcePath.parent() + Path(path)
-            let includedTemplate: String? = try? path.read()
+            var includedTemplate: String? = try? path.read()
```

Comment:
Well, I chose to do it like this so you can do things like `include('My.template.with.custom.dot.based.naming.scheme')` (which needs the `ejs` extension) and `include('Mytemplate.js')` (which does not have the `ejs` extension) which could be tricky otherwise.

If you really think it's better to check if the extension is `ejs` before importing (and limit the import function to `ejs` files), I'll change it of course, just let me know 🙂